bin/node/cli/src/rpc.rs

```diff
@@ -206,13 +219,30 @@ where
 	io.extend_with(BalancesApi::to_delegate(Balances::new(client.clone())));
 	io.extend_with(HeaderMMRApi::to_delegate(HeaderMMR::new(client.clone())));
 	io.extend_with(StakingApi::to_delegate(Staking::new(client.clone())));
+
+	let mut overrides = BTreeMap::new();
+	overrides.insert(
+		EthereumStorageSchema::V1,
```

Comment:
Need documents, especially for future update, old Schema should not be deleted for history blocks. (as we summarized)